eq-author-api/migrations/dropDatatypeFieldFromPipedValues.test.js

```diff
@@ -0,0 +1,91 @@
+const dropDatatypeFieldFromPipedValues = require("./dropDatatypeFieldFromPipedValues.js");
+
+describe("dropDatatypeFieldFromPipedValues", () => {
+  let questionnaire;
+  beforeEach(() => {
+    questionnaire = {
+      sections: [
+        {
+          randomNum: 100,
+          introductionTitle:
+            '<p>Intro content <span data-piped="answers"data-id="a06d223f-667a-45d4-85ce-22a9d8ff07d8"data-type="Currency">[Revenue 2018]</span></p>',
```

Comment:
There should be spaces between the attributes as that would match the data in the server.